src/server.ts

```diff
@@ -0,0 +1,23 @@
+import "config/env";
+import express, { Router } from "express";
+import { getLogger } from "./config/logger";
+import { setupGithubWebhooks } from "./webhook/setupGithubWebhooks";
+import { webhookReceiver } from "./webhook/webhook-receiver";
+import { Webhooks } from "./webhook/webhooks";
+const app = express();
+const port = 8082; 
```

Comment:
I'm not sure we can have the app run on port 8082. Because it runs on micros, I think it needs to be 8080.

Also, we don't necessarily need to do this as part of this PR because we're mainly focusing on the webhook aspect, but we should at least think about other places we're calling a Probot method in the app. For example, both main and worker call probot.start which is a method on Probot's server class that creates a httpServer. 

We can focus on adding methods that aren't webhook specific later but it might be good to keep them in mind now. I'm thinking refactoring this into a Server class might be a better approach. Instead of adding an additional script, I'm guessing we would want a similar implementation where we can call something like `webhookReceiver.start()` from main and worker and have everything 'just work'. 

For the simplicity of getting webhooks working without Probot, I'm happy for us to leave this as is for now but definitely think we need to modify this to completely replace Probot. Might be worthwhile setting up some time with some of the team to discuss how we want to tackle this.

Reply:
Definitely, this is going to be change a lot in future. Port 8080 is currently blocked by Probot

Reply:
there's no need to run a new server - just hook into the current one.